internal/provider/bootstrap.go

```diff
@@ -197,8 +198,16 @@ func oneTimeBootstrap(l logging.StandardLogger, c *providerConfig.Config, vpnSet
 	}
 
 	if utils.IsOpenRCBased() {
+		// Instead of creating an empty file, let's use the one generated by the k3s installer
+		content, err := os.ReadFile("/etc/init.d/k3s")
+		if err != nil {
+			l.Errorf("Failed to read init file: %s", err.Error())
+			return err
+		}
+
 		svc, err = openrc.NewService(
 			openrc.WithName(svcName),
+			openrc.WithUnitContent(string(content)),
```

Comment:
Should we also check if it's empty?

Reply:
Good point, I've added a check for this now